.bit/responses/Format.md

```diff
@@ -0,0 +1,35 @@
+---
+files: n/a
+stepType: checks
+scripts: test.parseimage.js
+week: 2
+step: 1
+name: Week 2 Step 1
+---
+
+**Week 1 Step 1** ⬤◯◯
+
+🕐 **Estimated completion:** 25-35mins
+
+## Learning GitHub
+### ✅ Task:
+
+**TASK DESCRIPTION.**
+- [ ] 
+
+## ❓ If you are confused
+
```

Comment:
Make sure to split the if you are confused part into sections, or else it's hard to read/navigate

Ex:

## ❓ If you are confused

### something
...dropdowns

### something
...dropdowns